### PR TITLE
Added method to evaluate element-wise arctan2

### PR DESCRIPTION
**Pull request by lindauruchurtu**
- Can be generalised to other functions.
- Also part of the PR, moving conanfile.py
- Minor version bump
- Amended `README.md` appropriately

### CODE REVIEW

File: src/cpp/polars/numc.cpp [hunk 1]
```diff
@@ -199,5 +199,15 @@ namespace polars {
             }
         }
 
+        arma::vec arctan(const arma::vec &x, const arma::vec &y) {
```
Comment (calvingiles) [inline src/cpp/polars/numc.cpp]:
```suggestion
        arma::vec arctan2(const arma::vec &x, const arma::vec &y) {
```

File: src/cpp/polars/Series.h [hunk 2]
```diff
@@ -151,6 +151,8 @@ namespace polars {
 
         Series tail(int n=5) const;
 
+        static Series arctan(const Series &lhs, const Series &rhs);
```
Comment (calvingiles) [inline src/cpp/polars/Series.h]:
```suggestion
        static Series arctan2(const Series &lhs, const Series &rhs);
```

File: src/cpp/polars/numc.cpp (hunk 1 above)
Comment (lindauruchurtu) [reply src/cpp/polars/numc.cpp]: Done.

File: src/cpp/polars/Series.h (hunk 2 above)
Comment (lindauruchurtu) [reply src/cpp/polars/Series.h]: Ditto.

File: conanfile.py [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
 class PolarsConan(ConanFile):
     name = "Polars"
-    version = "0.1.0"
+    version = "0.1.1"
```
Comment (calvingiles) [inline conanfile.py]: I think we discussed making this a minor version bump, not patch?

Comment (lindauruchurtu) [reply conanfile.py]: Gah. My bad. Amending.

**Review by calvingiles — APPROVED** [review]

**Pull request merged**